generators/app/templates/README.md

```diff
@@ -0,0 +1,42 @@
+# testcafe-browser-provider-<%= providerName %>
+[![Build Status](https://travis-ci.org/<%= githubUsername %>/testcafe-browser-provider-<%= providerName %>.svg)](https://travis-ci.org/<%= githubUsername %>/testcafe-browser-provider-<%= providerName %>)
+
+This is the **<%= providerName %>** browser provider plugin for [TestCafe](http://devexpress.github.io/testcafe).
+
+## Install
+
+```
+npm install -g testcafe-browser-provider-<%= providerName %>
```

Comment:
Let's not specify `-g` flag in example install, since it's dependends on installation. We need to update it in reporter generator as well (\cc @VasilyStrelyaev )